source/gui/settingsDialogs.py

```diff
@@ -268,15 +280,15 @@ class SettingsPanel(wx.Panel, DpiScalingHelperMixin, metaclass=guiHelper.SIPABCM
 
 	title=""
 	panelDescription=u""
+	helpId = ""

```

Comment:
Should we leave this out?

Reply:
I think I should move it into the `ContextHelpMixin`.